### PR TITLE
DO-NOT-MERGE: Greenkeeper eslint 2.13.0 autofix preview

### PR DESCRIPTION
**Pull request by Mardak**
@Standard8, do we want to enable any of these 3 now-autofixable rules? We probably don't want to actually merge this yet as 2.13.0 broke `extends` for us, but we can see what autofixed code looks like.

I needed to disable object-shorthand for karma as that's not es6.

### CODE REVIEW

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: test/karma/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "object-shorthand": [2, "never"]
```
Comment (Mardak) [inline test/karma/.eslintrc:3]: This actually runs object-shorthand and ensures we don't use shorthand (as opposed to just disabling).

File: shared/test/sdk_mock.js [hunk 2]
```diff
@@ -20,4 +20,3 @@
 
   window.OT.setLogLevel = function() {};
 })(window);
-
```
Comment (Mardak) [inline shared/test/sdk_mock.js]: Fyi, this was the only trailing space. All the other changes for this rule were for 2 lines -> 1 line.

File: shared/test/sdk_mock.js [hunk 3]
```diff
@@ -19,6 +19,4 @@
   };
 
   window.OT.setLogLevel = function() {};
-
 })(window);
-
```
Comment (Mardak) [inline shared/test/sdk_mock.js:24]: Fyi, this was the only trailing space. All the other changes for no-multiple-empty-lines rule were for 2 lines -> 1 line.

**Comment by Mardak** [comment]
Probably can skip this for now...

**Pull request closed without merging**